INTEGRATE.md

```diff
@@ -0,0 +1,31 @@
+# Local integration between Porta/System with Keycloak through Zync
+Install these 3 components to have a complete integration of managed SSO applications in Porta.
+
+## Install Porta
+Follow [the installation guide](https://github.com/3scale/porta/blob/master/INSTALL.md).
+
+## Install Zync
+Follow [the installation guide](https://github.com/3scale/zync/blob/master/INSTALL.md).
+
+## Install Keycloak
+To have a keycloak running locally, do `docker run --name keycloak -d -p 8080:8080 -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=p jboss/keycloak`
+
+Finally [configure Red Hat Single Sign-On](https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.4/html/api_authentication/openid-connect).
+
+## Run Zync
+From the folder where Zync is installed, run: `ZYNC_AUTHENTICATION_TOKEN=zynctoken bundle exec rails server -p 5000` to get zync running in the port 5000.
+
+## Run Porta
+These configurations are needed:
+  - In `config/rolling_updates.yml`, make sure to have `apicast_v2: true` and `apicast_oidc: true`.
+  - In `config/sandbox_proxy.yml`, make sure to have `apicast_registry_url: <%= ENV.fetch('APICAST_REGISTRY_URL', '<Insert a URL that contains policies in JSON. It can be am empty JSON>') %>`
+
+Run porta with `UNICORN_WORKERS=8 ZYNC_ENDPOINT=http://localhost:5000 ZYNC_AUTHENTICATION_TOKEN=zynctoken bundle exec rails server -b 0.0.0.0`
+
+### Run Porta's sidekiq
+Sidekiq Zync Worker process jobs from the `low` queue so do this: `ZYNC_ENDPOINT=http://localhost:5000 ZYNC_AUTHENTICATION_TOKEN=zynctoken bundle exec sidekiq -q low`
+
+## Configure 3scale to use Keycloak
+Follow the documentation to [configure 3scale](https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.4/html/api_authentication/openid-connect).
```

Comment:
_https://access.redhat.com/documentation/en-us/{DocInfoProductName}/{ProductVersion}/html/api_authentication/openid-connect_

Between docs you need full URLs with placeholders for product name ({DocInfoProductName}) and doc version ({ProductVersion}) used as variables, as in the following example: https://access.redhat.com/documentation/en-us/{DocInfoProductName}/{ProductVersion}/html-single/deployment_options/#apicast_policies[Policies]

Reply:
As spoken, it will be done as another task because it is for an internal audience and that way we are not blocked.
[THREESCALE-2075 - Autoupdate links of internal documentation](https://issues.jboss.org/browse/THREESCALE-2075)